welcome.md

```diff
@@ -0,0 +1,75 @@
+
+Governance is provided by the UMA Improvement Proposals (UMIPs) which allow new collateral types and price identifiers to be added to the system as well as new types of factory contract.
+
+
+**How UMA works**
+
+There are five main network actors on UMA. 
+
+1. Token sponsors
+2. Liquidator bots
+3. Disputer bots
+4. UMA Tokenholders
+5. Data Verification Mechanism (DVM)
+
+Token sponsors are individuals who lock collateral in a smart contract to mint synthetic tokens. 
+
+The value of the collateral in the smart contract is continually monitored off-chain by a robust network of Liquidator Bots. Liquidator Bots continuously monitor if a position is properly collateralized by referencing off-chain price feeds. Liquidator Bots are incentivized with rewards to identify and liquidate undercollateralized positions. If a position is liquidated by a liquidator bot there will be a 2-hour delay before the liquidation is finalized. 
+
+During the 2-hour delay, Dispute Bots are incentivized to continuously monitor contracts using UMA’s priceless financial contracts. Dispute Bots reference their own off-chain price feeds to determine if a liquidation was valid or in valid. If invalid, the dispute bot will dispute the liquidation which will call UMA’s oracle (i.e., the DVM) and the liquidated position will be pending until it is resolved by the DVM (48-hours later). 
+
+When a Dispute Bot calls the DVM, the liquidated position is reported to UMA Tokenholders where they will reference off-chain price feeds to report price information to the DVM. The DVM will aggregate UMA tokenholder votes and report the price of the asset on-chain and reward the Disputer Bot and penalize the Liquidator Bot (or vice versa if the liquidator was correct). Following the result from the DVM, the position will become liquidated or remain solvent. 
+
+
+**For Developers**
+
+If you are a developer interesting building with the UMA protocol, view this [explainer](/priceless-financial-contracts/priceless-contracts.md) of how UMA's priceless synthetic tokens wor and to learn more about UMA's synthetic tokens.
+
+Additionally, the [development guide](build-walkthrough/build-process) to learn how to launch your own expiring synthetic token on UMA.
+
+You can find additional reference materials, including a list of UMA's testnet and mainnet contracts [here](dev-ref/addresses.md)
+
+
+**For UMA Tokenholders**
+
+If you are interested in the UMA project token, please read [this section](uma-tokenholders/uma-holders.md) to learn more about the role of the UMA project token in the DVM and the rights and responsibilities of UMA project tokenholders.
+
+**Contacts**
+
+If you have any questions or want to connect with the UMA community, please check out our [community page](community/community-overview), join our [discord](https//discord.umaproject.org) and [discourse](https://discourse.umaproject.org)
+

```

Comment:
Unresolved issue raised by @Tommy1231232

"Did we align on this being in the welcome page? I'm open to it but I want to focus more on the What UMA does a bit more in that case. For example, We should explain our priceless concept a bit more where we talk about our priceless contract template. Or we put this in a different section after the level 2 of Priceless contracts and DVM"

Reply:
Personally I'm still in favor of the `Welcome` page being focused on how to navigate the site for different types of users. I do like the mission statement content, but don't think the `How UMA works` content really belongs on the first page.